### DISCUSSION TITLE
Reading a .obj file and creating a structured or unstructured grid

### DISCUSSION THREAD

**Discussion by ayenawine** [pyvista/pyvista#4350] [Q&A]
I have a *.obj file I created in Fusion 360 of a simple cylinder object.  How can I import this to pyvista and create a structured or unstructured grid of volume cells from it?  It appears to import as a surface mesh, so I am not sure how to get volume cells from it.  Can you do this with a .obj file or should I be using some other format?

Thanks!

**Comment by akaszynski** [comment]
I'd try using [delaunay_3d](https://docs.pyvista.org/api/core/_autosummary/pyvista.DataSetFilters.delaunay_3d.html). If that fails, you can also try [tetgen](https://pypi.org/project/tetgen/).
